server/src/models/Review.js

```diff
@@ -0,0 +1,46 @@
+const Model = require("./Model")
+
+class Review extends Model {
+  static get tableName() {
+    return "reviews"
+  }
+
+  static get relationMappings() {
+    const { Trail, User } = require("./index")
+
+    return {
+      trails: {
```

Comment:
Since both of these relations are `one to many`, we're saying that a Review belongs to one `trail`, so we should define the association as a singular `trail`

Reply:
Done